packages/main/src/themes/Badge.css

```diff
@@ -132,3 +132,15 @@
 	border-color: var(--ui5-badge-color-scheme-10-border);
 	color: var(--ui5-badge-color-scheme-10-color);
 }
+
+/* ---Slotted Badges--- */
+/* ui5-avatar - Badge icon styles */
+/* Make icon take full width minus padding.
+ ui5-avatar is the only component using an icon for badge,
+ therefore no addditional scoping is needed. */
+:host([slot="badge"]) ::slotted([ui5-icon][slot="icon"]) {
```

Comment:
Hello @SAP/ui5-webcomponents-topic-rd  I am adding you for review as there is a change (addition of styles) to the Badge.css